components/Ink/src/private/MDCLegacyInkLayer.m

```diff
@@ -490,24 +484,29 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   self = [super initWithCoder:aDecoder];
 
   if (self) {
-    _bounded = YES;
-    _inkColor = [UIColor colorWithWhite:0 alpha:(CGFloat)0.08];
+
     // Discard any sublayers, which should be the composite ripple and any active ripples
     if (self.sublayers.count > 0) {
       NSArray<CALayer *> *sublayers = [self.sublayers copy];
       for (CALayer *sublayer in sublayers) {
         [sublayer removeFromSuperlayer];
       }
     }
-    _compositeRipple = [CAShapeLayer layer];
-    _foregroundRipples = [NSMutableArray array];
-    _backgroundRipples = [NSMutableArray array];
-    [self addSublayer:_compositeRipple];
+    [self _initLegacyInkLayer];
   }
 
   return self;
 }
 
+- (void)_initLegacyInkLayer {
```

Comment:
Our code convention for common initializers is `common<#ClassName#>Init`. This helps reduce the risk of subclasses colliding with superclass common inits.

In this case then: `commonMDCLegacyInkLayerInit`

Reply:
Thanks for pointing out. Changed.